internal/godocfx/main.go

```diff
@@ -151,6 +151,7 @@ func process(mod indexEntry, tempDir, outDir string, print bool) error {
 	filter := []string{
 		"cloud.google.com/go/analytics",
 		"cloud.google.com/go/area120",
+		"cloud.google.com/go/gsuiteaddons/apiv1",
```

Comment:
```suggestion
		"cloud.google.com/go/gsuiteaddons",
```

So we don't have to update this for new versions in the future. The filter is prefix based.